core/src/main/java/org/apache/iceberg/deletes/Deletes.java

```diff
@@ -74,9 +75,24 @@ public static <T> CloseableIterable<T> markDeleted(
         });
   }
 
+  /**
+   * Returns the remaining rows (the ones that are not deleted), while counting the deleted ones.
```

Comment:
Please add the description for the parameters

Reply:
Fixed.